cwarhm/data_specific_processing/era5.py

```diff
@@ -0,0 +1,227 @@
+
+import netCDF4 as nc4
+from pathlib import Path
+import time, os
+
+
+def merge_era5_forcing(forcingPath, mergePath, years_str):
+    """ Combine separate surface and pressure level downloads
+    Creates a single monthly `.nc` file with SUMMA-ready variables for further processing. # Combines ERA5's `u` and
+    `v` wind components into a single directionless wind vector.
+
+    :param forcingPath: path to raw ERA5 surface and pressure level data
+    :param mergePath: path to save merged ERA5 data
+    :param year_str: start,end year string from control file (e.g., "2008,2013")
+    """
+
+    # pre proessing
```

Comment:
processing

Reply:
I intended to say "pre-processing" but changed anyway.